docs/software/advanced_networking.md

```diff
@@ -298,6 +299,52 @@ It is best suited for high traffic web sites and powers quite a number of the wo
 Website: <https://www.haproxy.org/>  
 Official documentation: <https://docs.haproxy.org/>
 
+## Avahi-Daemon
+
+Avahi is a Zeroconf networking implementation, offering DNS Service Discovery for a local network.  
+It is an open source implementation of DNS Service Discovery (DNS-SD) over Multicast DNS (mDNS), commonly known as and compatible with **Apple Bonjour**.
+
+Avahi is often used for a simple name resolution of a host on a local network (e.g. accessing a host via `<hostname>.local`).  
+More generally it enables programs to publish and discover services and hosts running on a local network.
+
+![Avahi logo](../assets/images/dietpi-software-advanced-networking-avahi-logo.png){: width="200" height="150" loading="lazy"}
+
+=== "Client access"
+
+    A client accessing a system running the Avahi daemon must have the mDNS tools Linux package installed (`libnss-mdns`). This can be achieved via
```

Comment:
```suggestion
    To listen and resolve hostnames distributed via mDNS, Linux systems need to have a package installed (`libnss-mdns`). This can be achieved via
```